### ISSUE TITLE
markdownlint-cli2-codequality: support custom file name

### ISSUE THREAD

**Issue by anb0s** [DavidAnson/markdownlint-cli2#112]
First thank you for the great tooling 👍 

I'm using the `markdownlint-cli2-junit` and also expected the name to be configurable for `markdownlint-cli2-codequality`:

```yml
# Use specific formatter(s) (only valid at root)
outputFormatters:
# https://github.com/DavidAnson/markdownlint-cli2/tree/main/formatter-summarize
- - markdownlint-cli2-formatter-summarize
  - byFileByRule: true
# https://github.com/DavidAnson/markdownlint-cli2/tree/main/formatter-pretty
- - markdownlint-cli2-formatter-pretty
  - appendLink: true
# https://github.com/DavidAnson/markdownlint-cli2/tree/main/formatter-junit
- - markdownlint-cli2-formatter-junit
  - name: markdownlint-report.xml
# https://github.com/DavidAnson/markdownlint-cli2/tree/main/formatter-codequality
- - markdownlint-cli2-formatter-codequality
  - name: markdownlint-quality.json
```

It creates the file `markdownlint-cli2-codequality.json`

I know it's documented, but would be good to have it configurable :)

**Comment by anb0s** [comment]
> I made a PR to add this capability. One thing of note, in your "example" you specified a filename that ends in .XML, but regardless the file will be in json format, as codequality appears to require json, with XML not being an option.

Thank You! Yes, it was an "copy and paste" issue during my first try :) now changed to json.

**Comment by DavidAnson** [comment]
Fixed in latest published version `0.0.4` (and possibly the previous).
